examples/plotting/spans_and_strips.py

```diff
@@ -1,3 +1,11 @@
+''' This example demonstrates the use of ``hspan``, ``wspan``, and ``vstrip``.
+It also demonstrates how the default ``HoverTool`` interacts with these glphs.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.figure.hspan, bokeh.plotting.figure.hstrip, bokeh.plotting.figure.vstrip
+    :keywords: strips, hover tool
+'''
```

Comment:
We try to end docstrings with a blank line
```suggestion
    :keywords: strips, hover tool
    
'''
```

Reply:
I will add the newline.